### PR TITLE
Take environment variables for database credentials

### PR DESCRIPTION
**Pull request by eternaltyro**

### CODE REVIEW

**Comment by eternaltyro** [comment]
@itskshitiz321 @JorgeMartinezG @omranlm please help validate my changes including syntax and functionality. TIA for the help.

**Review by itskshitiz321 — DISMISSED** [review]
These config files are defined in API/__init__.py  needs to be inside src/config.py

**Comment by eternaltyro** [comment]
@itskshitiz321 @JorgeMartinezG made the changes you suggested. Please review.  cc: @ramyaragupathy

**Comment by eternaltyro** [comment]
Syntax errors fixed.

**Review by itskshitiz321 — CHANGES_REQUESTED** [review]

File: API/changesets/routers.py [hunk 1]
```diff
@@ -101,13 +101,11 @@ def get_changesets(params: FilterParams):
...
         """
 
-    db_params = dict(config.items("PG"))
-    conn = psycopg2.connect(**db_params)
-
-    cur = conn.cursor(cursor_factory=DictCursor)
-    cur.execute(query)
-
-    result = cur.fetchall()
+    db_params = get_db_connection_params()
```
[the remark this answers is not in the document]
Comment (eternaltyro) [reply API/changesets/routers.py:105]: @itskshitiz321 done.

File: API/countries/routers.py [hunk 2]
```diff
@@ -11,31 +11,30 @@
...
-            FROM t1 GROUP BY name, tags)
-        SELECT json_build_object('type',
-            'FeatureCollection',
-            'features',
-            json_agg(ST_ASGEOJSON(t2.*)::json))
-        FROM t2
-        """
-    )
-    result = cur.fetchall()[0][0]
+    db_params = get_db_connection_params()
```
[the remark this answers is not in the document]
Comment (eternaltyro) [reply API/countries/routers.py:15]: @itskshitiz321 done.

**Comment by eternaltyro** [comment]
Added missing imports.

**Pull request merged**